src/service/gateway.proto

```diff
@@ -36,6 +36,8 @@ message gateway_sc_is_active_resp_v1 {
   bytes sc_id = 1;
   bytes sc_owner = 2;
   bool active = 3;
+  uint64 sc_expiry_at_block = 4;
```

Comment:
why prefix with `sc_` here if `active is not?

Reply:
the way i have it is any field derived from the SC itself is prefixed with the sc, and active in this case is not taken from the SC but signals the SC does exist

Reply:
Got it.. good explanation, thanks